src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1113,8 +1113,19 @@ public void updateInstallation(int id, String json) {
                 logger.debug("Request {} {}", request.getMethod(), request.getUrl());
             }
             return client.executeRequest(request, handler);
```

Comment:
@stephenc what did you need to check wrt compatibility, I would be for removing the try catch and bumping major version.